os/arch/arm/src/s5j/s5j_i2s.c

```diff
@@ -1592,6 +1596,63 @@ static uint32_t i2s_txdatawidth(struct i2s_dev_s *dev, int bits)
 
 /***************************************************************************
 ****************************************************************************/
+static int i2s_pause(struct i2s_dev_s *dev)
+{
+	struct s5j_i2s_s *priv = (struct s5j_i2s_s *)dev;
+	irqstate_t flags;
+	struct s5j_buffer_s *bfcontainer;
+	DEBUGASSERT(priv);
+
+#ifdef I2S_HAVE_TX_P
+	flags = irqsave();
+	s5j_dmastop(priv->txp.dma);
+	modifyreg32(priv->base + S5J_I2S_CON, I2S_CR_FTXURINTEN, 0);
+	modifyreg32(priv->base + S5J_I2S_CON, I2S_CR_TXDMACTIVE, 0);
```

Comment:
This is not pause. the dma registers should be modified for dma pause. First of all not sure if i2s_pause can be generic method to be added (leave the decision to Ivan).